test/unit/argparse_test.lua

```diff
@@ -392,6 +392,21 @@ g.test_replication_quorum_opts = function()
         replication_synchro_quorum = 'N/3+5',
     })
 
+    utils.file_write(
```

Comment:
Is it possible to add some custom parameter e.g. "number|boolean" and check that it's also parsed correctly?